lib/zip.js

```diff
@@ -27,50 +34,89 @@ const afs = require('./fs'),
  */
 exports.unzip = function unzip(file, dest, opts, finished) {
 	try {
-		const zip = new AdmZip(file),
-			zipEntries = zip.getEntries(),
-			len = zipEntries.length,
-			Utils = require('adm-zip/util'),
-			visitor = opts && opts.visitor || function () {},
+		const visitor = opts && opts.visitor || function () {},
 			overwrite = opts && Object.prototype.hasOwnProperty.call(opts, 'overwrite') ? !!opts.overwrite : true,
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/zip.js line 3...
  
  
  DangerID: danger-id-Danger;
  File: lib/zip.js;
  Line: 38;
-->

- :warning: lib/zip.js line 38 – 'overwrite' is assigned a value but never used. (no-unused-vars)